assets/styles.css

```diff
@@ -0,0 +1,200 @@
+* {
+  box-sizing: border-box;
+  font-family: "Josefin Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav li,
```

Comment:
Just a little note to be cautious using purely element selectors like this - there's absolutely nothing wrong with it technically (and there's no need to change it here) but as your projects start to grow in size, you probably want to stick more with more specific selectors, like the class selectors and ID selectors. These just avoid styles being inadvertently applied to other elements in the page with the same element type as you have to specifically specify the class name. Obviously this is unlikely to happen here, but when your projects grow (and especially when working on group projects), it will be much more likely.